Web/keywords/features/home_features.robot

```diff
@@ -0,0 +1,5 @@
+*** Keywords ***
+
```

Comment:
จิงๆ ไม่ต้องเว้น 1 บรรทัดหลัง keywords คับ แต่ว่าปอนด์ทำมาทุกไฟล์เป็น style เดียวกันเลยไม่ต้องแก้ก็ได้ค่ะ
แต่ถ้าได้ทำงาน กับ doppio automate team คือต้องลบนะคะ
pattern จะเป็นแบบนี้

*** Keywords ***
Search Product And Add Product to Cart
    home_page.Search Product 
    home_page.Add Product to Cart
              <<<< เว้น 1 บรรทัด
keyword2